Jenkinsfile

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env groovy
```

Comment:
CentOS CI unfortunately doesn't support Jenkinsfiles like this one, so it can be removed.

Reply:
@msrb So, is there a specific shebang we need to use for Jenkins file?